Sprint-2/extend/format-time.js

```diff
@@ -2,9 +2,15 @@
 // Your task is to write tests for as many different groups of input data or edge cases as you can, and fix any bugs you find.
 
 function formatAs12HourClock(time) {
-  const hours = Number(time.slice(0, 2));
+  const hours = Number(time.slice(0, 2)); // Gets the hour as a number 
+  const minutes = time.slice(3) // Gets the minutes 
+
+  if(hours === 0){                // Checks for midnight hour (00:00)
+    return `12:${minutes} am`
+  }
+
```

Comment:
What should "12:01" be converted to?

Reply:
I've fixed the bug by adding another checkpoint for 12:01 pm